src/index.ts

```diff
@@ -293,6 +293,26 @@ const getChainspecName = (chain: Chain | string, id: number) => {
   return `${stripChainspecJsonName(chainName)}-${id}.json`;
 };
 
+/**
+ * Calculate Collator's Stake
+ *
+ * @param chain
+ */
+const generateCollatorStake = (chain: string) => {
+  console.log(chain);
+  if (chain === 'dev-local') {
+    return 64000;
+  } else if (chain === 'agung-local') {
+    return 64000;
+  } else if (chain === 'krest-local') {
+    return 2 * 1000 * 1000000000000000000;
```

Comment:
Why is `krest-local` the one breaking out of the line? :upside_down_face:

Reply:
Hahahaha, because the min collator stake in the Krest network is 1000 * peaq.